src/ipc-handlers.ts

```diff
@@ -993,81 +992,39 @@ function promiseExec( command: string, options: ExecOptions = {} ): Promise< voi
 	} );
 }
 
-export async function openTerminalAtPath(
-	_event: IpcMainInvokeEvent,
-	targetPath: string,
-	{ wpCliEnabled }: { wpCliEnabled?: boolean } = {}
-) {
+export async function openTerminalAtPath( _event: IpcMainInvokeEvent, targetPath: string ) {
 	const platform = process.platform;
-	const cliPath = nodePath.join( getResourcesPath(), 'bin' );
-	const exePath = app.getPath( 'exe' );
-	const appDirectory = app.getAppPath();
-	const appPath = ! app.isPackaged ? `${ exePath } ${ appDirectory }` : exePath;
 
 	if ( platform === 'darwin' ) {
 		const initScriptSteps = [];
-
-		if ( wpCliEnabled ) {
-			initScriptSteps.push(
-				`export PATH=\\"${ cliPath }\\":$PATH`,
-				`export STUDIO_APP_PATH=\\"${ appPath }\\"`
-			);
-		}
-
 		const escapedPath = targetPath.replace( /"/g, '\\"' );
 		initScriptSteps.push( `cd \\"${ escapedPath }\\"`, 'clear' );
 
 		const userData = await loadUserData();
 		const preferredTerminal = userData.preferredTerminal || DEFAULT_TERMINAL;
 
-		if ( preferredTerminal === 'warp' ) {
-			return promiseExec( `open -a Warp "${ targetPath }"` );
-		} else if ( preferredTerminal === 'ghostty' ) {
-			return promiseExec( `open -a Ghostty "${ targetPath }"` );
-		} else if ( preferredTerminal === 'iterm' ) {
-			return promiseExec( `osascript << END
-tell application "iTerm"
-    activate
-    create window with default profile
-    tell current session of current window
-        write text "${ initScriptSteps.join( ';' ) }"
-    end tell
-end tell
-END` );
-		} else {
-			return promiseExec( `osascript << END
-activate application "Terminal"
-tell application "Terminal"
-    do script "${ initScriptSteps.join( ';' ) }"
-end tell
-END` );
-		}
+		const terminalApps = {
+			warp: 'Warp',
+			ghostty: 'Ghostty',
+			iterm: 'iTerm',
+			terminal: 'Terminal',
+		};
+		const appName = terminalApps[ preferredTerminal ] || 'Terminal';
+		return promiseExec( `open -a ${ appName } "${ escapedPath }"` );
 	} else if ( platform === 'win32' ) {
 		const userData = await loadUserData();
 		const preferredTerminal = userData.preferredTerminal;
 		const defaultShell = process.env.ComSpec || 'cmd.exe';
-		const env = wpCliEnabled
-			? { PATH: `${ cliPath };${ process.env.PATH }`, STUDIO_APP_PATH: appPath }
-			: {};
 
 		if ( preferredTerminal === ( 'warp' as SupportedTerminal ) ) {
```

Comment:
```suggestion
		if ( preferredTerminal === 'warp' ) {
```
Not the fault of this PR, but there's no need to use the `as` operator here.

Reply:
Adjusted in https://github.com/Automattic/studio/pull/1426/commits/fa54af47d3ac63e209167617c652619d7e91a33c